controllers/cluster_scripts/cloud_init.tmpl

```diff
@@ -33,21 +43,21 @@ write_files:
     CSI_NODE_PATH=/root/csi-node.yaml {{- end }}
 
     vmtoolsd --cmd "info-set guestinfo.postcustomization.networkconfiguration.status in_progress"
-    hostname "{{ .MachineName }}"
-    echo "::1         ipv6-localhost ipv6-loopback" >/etc/hosts
-    echo "127.0.0.1   localhost" >>/etc/hosts
-    echo "{{ .MachineName }}" >/etc/hostname
```

Comment:
we should keep this entry ` echo "{{ .MachineName }}" >/etc/hostname`

Reply:
Interesting; even without this entry, in my development cluster, `/etc/hostname` contains the machine name :thinking:

Reply:
I think cloud-init sets `/etc/hostname`, based on this configuration:
https://github.com/mesosphere/cluster-api-provider-cloud-director/blob/66ecf82670254cafae0e84f9fc30d6fa454e5e28/controllers/cluster_scripts/cloud_init.tmpl#L84

Reply:
Thanks. That makes sense.